src/lib/app-provider/tx/clearAdmin.ts

```diff
@@ -15,7 +15,13 @@ export const useClearAdminTx = (contractAddress: ContractAddr) => {
   const { address, getCosmWasmClient } = useWallet();
   const queryClient = useQueryClient();
   const fabricateFee = useFabricateFee();
-  const clearAdminFee = fabricateFee(CLEAR_ADMIN_GAS);
+  const {
+    chainConfig: {
+      features: { wasm },
+    },
+  } = useCelatoneApp();
+  const gas = wasm.enabled ? wasm.storeCodeMaxFileSize : 0;
```

Comment:
1. clearAdminGas
2. is 0 correct for default?

Reply:
if wasm is not enabled, clear admin should not work so I set to 0